### DISCUSSION TITLE
Cursor randomly freezing on lazer only

### DISCUSSION THREAD

**Discussion by puqueumiamo** [ppy/osu#22684] [Q&A]
### Type

Game behaviour

### Bug description

while testing the osu!lazer new features i noticed that my cursor is randomly freezing sometimes. I tried restarting the game for it to not work. Im using the latest wacom driver for the CTH-480 without W. Ink. What is happening? The cursor behave normally on stable. And yes. the same applies to gameplay. And my tablet input on lazer is turned off cuz i prefer the wacom drivers smoothing.

### Screenshots or videos

https://youtu.be/DTXG03da2p8

### Version

2023.207.0

### Logs

[database.log](https://github.com/ppy/osu/files/10774053/database.log)
[input.log](https://github.com/ppy/osu/files/10774054/input.log)
[network.log](https://github.com/ppy/osu/files/10774055/network.log)
[performance.log](https://github.com/ppy/osu/files/10774056/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10774057/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10774058/updater.log)

**Comment by puqueumiamo** [comment]
Sorry for taking your time

**Comment by bdach** [comment]
Can you attach another video of this with frame graph shown? (press ctrl-f11 twice, should be in the bottom right corner)

**Comment by puqueumiamo** [comment]
Here: https://youtu.be/wz1-z4fpHWI
Logs:
[database.log](https://github.com/ppy/osu/files/10775158/database.log)
[input.log](https://github.com/ppy/osu/files/10775159/input.log)
[network.log](https://github.com/ppy/osu/files/10775160/network.log)
[performance.log](https://github.com/ppy/osu/files/10775161/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10775162/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10775163/updater.log)

**Reply by puqueumiamo** [reply]
Again, sorry for taking your time.

**Reply by bdach** [reply]
I'm... not sure what this issue is. Does turning off tablet in lazer options, or disabling "high precision mouse", do anything to fix?

**Reply by puqueumiamo** [reply]
> I'm... not sure what this issue is. Does turning off tablet in lazer options, or disabling "high precision mouse", do anything to fix?

When turning off "high precision mouse"(AKA raw input) it goes back to normal. But i do play with it on stable and it runs normally.
